src/sentry_config.py

```diff
@@ -0,0 +1,30 @@
+from configparser import ConfigParser
+
+import requests
+
+SENTRY_CONFIG_API_URL = (
+    "https://api.github.com/repos/{owner}/.sentry/contents/sentry_config.ini"
+)
+SENTRY_CONFIG_RAW_URL = (
+    "https://raw.githubusercontent.com/{owner}/.sentry/main/sentry_config.ini"
+)
+
+
+def fetch_dsn_for_github_org(org: str) -> str:
+    cp = ConfigParser()
+    api_url = SENTRY_CONFIG_API_URL.replace("{owner}", org)
```

Comment:
This looks pretty silly; any other recommendation?

Reply:
could use `.format(owner=owner)` or an f-string inline :shrug:

Reply:
That's what I was looking for! I missed the comment. Maybe on next PR.